docs/index.asciidoc

```diff
@@ -128,13 +128,15 @@ is used. AES Algorithms generally use a 16-byte IV:
 
 The key to use for encryption and decryption operations.
 
-NOTE: If you encounter an error message at runtime containing the following:
+[NOTE]
+============
+If you encounter an error message at runtime containing the following:
 
-"java.security.InvalidKeyException: Illegal key size: possibly you need to install 
-Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files for your JRE"
+`java.security.InvalidKeyException: Illegal key size: possibly you need to install 
+Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files for your JRE`
 
 Please read the following: https://github.com/jruby/jruby/wiki/UnlimitedStrengthCrypto
-
+============
 
```

Comment:
Thanks for this formatting fix, @adoerrES.  I tested your fix locally and it looks good: 
<img width="863" alt="Screen Shot 2022-02-10 at 5 47 10 PM" src="https://user-images.githubusercontent.com/35154725/153511020-cbeb5b89-7204-4c57-ace4-89df54f6c478.png">